pkg/apis/apis.go

```diff
@@ -12,6 +14,8 @@ var AddToSchemes runtime.SchemeBuilder
 func AddToScheme(s *runtime.Scheme) error {
 	// add openshift specific resource
 	AddToSchemes = append(AddToSchemes, userv1.AddToScheme)
+	AddToSchemes = append(AddToSchemes, templatev1.AddToScheme)
```

Comment:
Hm.. Why do we need templates and project schemes here?

Reply:
We are using it here [link](https://github.com/codeready-toolchain/member-operator/pull/40/files/e57cdfdbf3c8fc3dc72e46d8b6efe0f3863fc243#diff-cffea82f7fdc7427b8c9f0d0fdd55aedR13) in NSTemplateSetCtrl.  We need to add all scheme which are used in operator.  Let me know if you asking something more.